### PR TITLE
Canada: Updated region geometries and county names

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1475 and https://github.com/whosonfirst-data/whosonfirst-data/issues/1487.

To do:
  - [x] Update geometry of Quebec
  - [x] Update geometry of Nunavut
  - [x] Update `name:` properties in county records of Nunavut

Geometries do change, but since we're storing the existing geometries are `reversegeo:` alt geoms, no need to PIP. Can merge once approved.

### CODE REVIEW

**Comment by nvkelso** [comment]
Please ensure the quattroshapes geom is marked for revGeo.

**Comment by stepps00** [comment]
Screen Shot 2019-03-29 at 2 24 55 PM
Screen Shot 2019-03-29 at 2 24 43 PM

This PR includes new geometries for Nunavut and Quebec regions in Canada, as well as updated `name:` properties in the county records of Nunavut. cc @thisisaaronland @burritojustice

File: data/115/886/906/3/1158869063.geojson [hunk 1]
```diff
@@ -17,10 +17,18 @@
     "mz:hierarchy_label":1,
     "mz:is_current":1,
     "name:eng_x_preferred":[
-        "Baffin"
+        "Qikiqtaaluk"
     ],
     "name:eng_x_variant":[
-        "Baffin"
+        "Baffin",
```
Comment (nvkelso) [inline data/115/886/906/3/1158869063.geojson]: Are these `Baffin` variants historic names?

Comment (nvkelso) [reply data/115/886/906/3/1158869063.geojson]: No, it's a semi-official variant. Fun times: https://en.wikipedia.org/wiki/Qikiqtaaluk_Region.

**Pull request merged**